the-collab-lab/tcl-77-smart-shopping-list
# moving our auth/unauth routes into new auth/unauth sub directories

_For an example of how to fill this template out, [see this Pull Request](https://github.com/the-collab-lab/tcl-3-smart-shopping-list/pull/44)._

## Description



With the mention of moving some files in the views of in PR #27. I added the `authenticaed` and `unauthenticated` subdirectories in views and just moved `ManageList`, `List` and `PageNotFound` in to their proper directories.

## Related Issue



There is no current issue.

## Type of Changes

Use one or more labels to help your team understand the nature of the change(s) you’re proposing. E.g., `bug fix` or `enhancement` are common ones.


## Testing Steps / QA Criteria



- There shouldn't be any functional changes, but please check to make sure it works on yalls computers too! 😅
